modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -44,88 +44,97 @@ The following table shows the available feature flags, a description of their fu
 [%header%autowidth.stretch,cols="a,a"]
 |===
 |Feature Flag | Description
-<.^|`HONOUR_RESERVED_PROPERTIES_PROPERTY`
-|When enabled, reserved properties such as `app.name` can't be overridden by global properties (overrides are ignored).
+<.^|`mule.batch.fixed.aggregator.transaction.record.buffer`
+|When enabled, batch aggregators with fixed size aggregators commit only when a full block is processed.
 
 *Available Since*
 
 * 4.4.0
-* 4.3.0-202102
+
+* 4.3.0-202103
+
+* 4.2.2-202103
 
 *Enabled by Default Since*
 
-* 4.4.0
-* 4.3.1
+* Not enabled by default in any Mule version
 
 *Issue ID*
 
-* MULE-17659
-<.^|`HANDLE_SPLITTER_EXCEPTION_PROPERTY`
-|When enabled, DataWeave correctly handles internal exceptions while splitting a payload, preventing subsequent serialization errors.
+* MULE-19218
+<.^|`mule.compute.connection.errors.in.stats`
+|When enabled, connection errors are computed as part of alerts triggering.
 
 *Available Since*
 
 * 4.4.0
 
+* 4.3.0-202103
+
 *Enabled by Default Since*
 
 * 4.4.0
 
 *Issue ID*
 
-* DW-383
-<.^|`BATCH_FIXED_AGGREGATOR_TRANSACTION_RECORD_BUFFER_PROPERTY`
-|When enabled, batch aggregators with fixed size aggregators commit only when a full block is processed.
+* MULE-19020
+<.^|`mule.enable.default.errorhandler.not.rollback.incorrect.tx`
+|When enabled, the default error handler added by the Runtime doesn't rollback a transaction if it doesn't correspond.
 
 *Available Since*
 
-* 4.4.0
-* 4.3.0-202103
-* 4.2.2-202103
+* 4.4.0-202201
+
+* 4.3.0-202201
 
 *Enabled by Default Since*
 
-* Not enabled by default in any Mule version
+* 4.5.0
+
+* 4.4.1
+
+* 4.3.1
 
 *Issue ID*
 
-* MULE-19218
-<.^|`COMPUTE_CONNECTION_ERRORS_IN_STATS_PROPERTY`
-|When enabled, connection errors are computed as part of alerts triggering.
+* MULE-19919
+<.^|`mule.dw.remove_shadowed_implicit_inputs`
+|When enabled, DataWeave removes implicit inputs when a variable with the same name is declared at the root level.
 
 *Available Since*
 
-* 4.4.0
-* 4.3.0-202103
+* 4.4.0-202111
 
 *Enabled by Default Since*
 
 * 4.4.0
-* 4.3.1
 
 *Issue ID*
 
-* MULE-19020
-<.^|`TO_STRING_TRANSFORMER_TRANSFORM_ITERATOR_ELEMENTS_PROPERTY`
-|When enabled, managed cursor iterators transformed to strings show the representation of the elements instead of generic value `org.mule.runtime.core.internal.streaming.object.-ManagedCursorIteratorProvider$ManagedCursorIterator@######`.
+* DW-893
+<.^|`mule.enable.byteBuddy.objectCreation`
+|When enabled, the Objects factories are created with Byte Buddy instead of CGLIB.
```

Comment:
If I am understanding this correctly, Objects factories should be possessive. Also, Objects should be lowercase - "object's factories."